test/data/ansible/check_ostree.yaml

```diff
@@ -277,6 +277,7 @@
     - name: run ubi8 image
       command: podman run ubi8-minimal:latest cat /etc/redhat-release
       register: podman_result
+      become: yes
```

Comment:
This test was failing due to a permissions error so I am having it run as root. @henrywang Is there any issue with doing this?